test/concerns/content_test.rb

```diff
@@ -0,0 +1,39 @@
+require "test_helper"
+
+class ContentTest < ActiveSupport::TestCase
+  test "alias _list_item_title" do
+    book = Book.create(title: "The Art of War")
+    assert_equal("The Art of War", book._list_item_title)
+  end
+
+  test "#canonical_url" do
+    book = Book.create(title: "The Art of War")
+    if Rails.env.production?
```

Comment:
@nkumeiko 
Please, remove this `Rails.env.production?`. `Rails.env` is always `test`.